charts/refinery/values.yaml

```diff
@@ -289,6 +289,13 @@ autoscaling:
   annotations: {}
     # my-annotation: some value
 
+rollout:
+  # Disable with `rollingUpdate: {}` if using a different `strategy``:
+  rollingUpdate:
+    maxUnavailable: 1
+    maxSurge: 1
```

Comment:
To make sure this change is backwards compatible, let's default the value to `{}`.  We can leave the 2 entries underneath commented out.